.github/workflows/release.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
       CHAINLOOP_ROBOT_ACCOUNT: ${{ secrets.CHAINLOOP_ROBOT_ACCOUNT }}
       CONTAINER_IMAGE_CP: ghcr.io/chainloop-dev/chainloop/control-plane:${{ github.ref_name }}
       CONTAINER_IMAGE_CAS: ghcr.io/chainloop-dev/chainloop/artifact-cas:${{ github.ref_name }}
+      GH_TOKEN: ${{ github.token }}
```

Comment:
Are you sure you need to expose the env variable? I believe it also works with GITHUB_TOKEN https://cli.github.com/manual/gh_help_environment

Reply:
I've added the env variable that was recommended by the CLI when running it and it stated that was GH_TOKEN :/

Reply:
what do you mean that it was recommended? A warning message or smth?

I can see this in their docs

> GH_TOKEN, GITHUB_TOKEN (in order of precedence): an authentication token for github.com API requests. Setting this avoids being prompted to authenticate and takes precedence over previously stored credentials.

Reply:
>what do you mean that it was recommended? A warning message or smth?

Precisely it was this error message: https://github.com/chainloop-dev/chainloop/actions/runs/9079530767/job/24948940556

>gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example:
  env:
    GH_TOKEN: ${{ github.token }}

Reply:
Interesting, thanks for the clarification.

FTR for the future, the error is 

> gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example: